integration-cli/docker_cli_help_test.go

```diff
@@ -51,19 +52,14 @@ func (s *DockerSuite) TestHelpTextVerify(c *check.C) {
 		helpCmd := exec.Command(dockerBinary, "help")
 		helpCmd.Env = newEnvs
 		out, ec, err := runCommandWithOutput(helpCmd)
-		if err != nil || ec != 0 {
```

Comment:
why not remove the whole `runCommandWithOutput`/`exec.Command` above and just use `dockerCmd`? Also get rid of status code check, no need for it